frontend/src/app/App.tsx

```diff
@@ -118,12 +120,20 @@ const App = () => {
               )}
             />
             <ProtectedRoute
-              path="/results/:page?"
+              path="/results/"
+              render={() => <CleanTestResultsList />}
+              requiredPermissions={appPermissions.results.canView}
+              userPermissions={data.whoami.permissions}
+              exact={true}
+            />
+            <ProtectedRoute
+              path="/results/:page"
```

Comment:
Another option here instead of passing `exact` through is to put the route with `:page` first, so that the more specific route gets matched first:
https://reactrouter.com/web/guides/primary-components

Reply:
yeah but that would be order dependent, meaning any harmless reorder might cause a functional issue

Reply:
made the change